schemas/compat-data.schema.json

```diff
@@ -63,6 +63,24 @@
             "required": ["type", "name"]
           }
         },
+        "implementation_url": {
```

Comment:
To safe space and to be consistent with `spec_url`, I would suggest a shorter name like `implement_url` or `impl_url`.